src/components/CodeSnippet.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="code-snippet">
-    <CodeHighlighting class="code-snippet__code" :value="snippet" :language="language" :show-line-numbers="showLineNumbers" />
+    <p-code-highlight class="code-snippet__code" :text="snippet" :lang="language" :show-line-numbers="showLineNumbers" />
     <PButton size="sm" class="code-snippet__button" @click="copy">
       Copy <p-icon icon="DuplicateIcon" />
     </PButton>
```

Comment:
Should this use the CopyableWrapper component? If not, suggest same style treatments for the button as proposed in that component.

Reply:
this whole component is going to be removed in part 2, I only made this change because this PR removed CodeHighlighting